src/lib/mod.rs

```diff
@@ -1,15 +1,15 @@
 extern crate rand;
 
-mod life_106;
+mod parsers;
 
 use std::fs::File;
 use std::io::Read;
-use self::life_106::Parser;
 
 pub struct GameOfLife {
     pub board: Vec<Vec<bool>>,
     pub width: usize,
     pub height: usize,
+    pub generation: u64,
```

Comment:
I think keeping track of the generation is not necessary.

Reply:
It was mostly for debugging at first but I thought it might be worth keeping around as an option, let me know what you think about another flag `--show-generation` or another mechanism for giving a user this option.